app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TweetController < ApplicationController
+	
+	get '/tweets' do 
+		@tweets = Tweet.all 
+		auth_erb('tweets/tweets')
+	end
+	
+	get '/tweets/new' do 
+		auth_erb('tweets/create_tweet')
+	end
+	
+	post '/tweets/new' do 
+		tweet = Tweet.new(:content => params[:content])
+		if valid_tweet?(tweet)
+			tweet.save
+			current_user.tweets << tweet
+			current_user.save
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```

or alternatively

```
@tweet = current_user.tweets.create(content: params[:content])
```